addon/services/stripev3.js

```diff
@@ -43,8 +43,13 @@ export default Service.extend({
       this.configure();
     }
   },
-
-  load(publishableKey = null) {
+ 
+  unload() {
+    this.set('didConfigure', false);
+    this.set('didLoad', false);
+  },
+  
+  load(publishableKey = null, params = null) {
```

Comment:
can we set the `params` on `this` so that they are available in tests to assert that the correct params were passed through? Unlike `publishableKey`, I don't think they need to be wrapped in an `if` before setting, because no params are taken from default configuration.